bigbio/hub/hub_repos/medparasimp/medparasimp.py

```diff
@@ -0,0 +1,250 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Paragraph-level Simplification of Medical Texts ("MedParaSimp") is a
+dataset that contains pairs of technical medical abstracts from
+biomedical systematic reviews (taken from the Cochrane Library)
+and their corresponding plain-language summarizations (PLS).
+The PLS's were created by the authors of the original abstracts.
+The dataset was obtained by scraping the Cochrane Library website.
+"""
+
+from typing import Dict, List, Tuple
+
+import datasets
+
+from bigbio.utils import schemas
```

Comment:
The schema needs to be importend from `.bigbiohub` rather than bigbio (which is not available when loading from the HF hub)